### PR TITLE
kmal/issue forms - adding new issue forms to templates

### PR DESCRIPTION
**Pull request by kmal808**
Utilizing GitHubs new [Issue Forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) 

PR adds the following:

👀 `BUG-REPORT.yml`:

https://user-images.githubusercontent.com/20919083/215253949-364be3dc-b004-4293-a9ce-71845628e90b.mov

🪄 `FEATURE-REQUEST.yml`

https://user-images.githubusercontent.com/20919083/215254013-d0afdca0-441e-43f8-a635-5390c33bfae1.mov

These along with others coming soon will be used as issue templates for the repo. Along with the issue forms, additional files being added are `config.yml` and `CONTRIBUTING.md`.

### CODE REVIEW

**Review by tyliec — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Code for Hawaii
+    url: https://github.com/CodeforHawai/discussions
```
Comment (tyliec) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    url: https://github.com/CodeforHawaii/discussions
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    about: Start a conversation here
+  - name: Code for Hawaii website
+    url: https://codeforhawaii.org
+    about: Contact us through our website form
```
Comment (tyliec) [inline .github/ISSUE_TEMPLATE/config.yml]: We have a form? 😃

Comment (avenmia) [reply .github/ISSUE_TEMPLATE/config.yml]: We do not lol it got stripped out because it didn't work

**Review by avenmia — APPROVED** [review]
lgtm! :shipit:

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (kmal808) [reply .github/ISSUE_TEMPLATE/config.yml]: Forgot about that. My b. 
Should we/I get one working or nah?

**Comment by kmal808** [comment]
🚢 🚀 🫡
PR forms & templates are next on the "admin" list
Any/all input from @CodeforHawaii/hawaii-zoning-atlas would be well received

**Pull request merged**